deps/db/deps.edn

```diff
@@ -1,9 +1,10 @@
 {:deps
  ;; External deps should be kept in sync with https://github.com/logseq/nbb-logseq/blob/main/bb.edn
  {datascript/datascript {:git/url "https://github.com/logseq/datascript" ;; fork
-                         :sha     "6d9b5f0db22d960ddf4870503bc181bff9a50c5d"}
+                         :sha     "21fc7880c7042fb1d9086135d162ea7a91681f89"}
   com.cognitect/transit-cljs   {:mvn/version "0.8.280"}
-  cljs-bean/cljs-bean          {:mvn/version "1.5.0"}}
+  cljs-bean/cljs-bean          {:mvn/version "1.5.0"}
+  logseq/graph-parser          {:local/root "../../deps/graph-parser"}}
```

Comment:
@tiensonqin This creates a circular dependency as graph-parser already depends on db. gp-config and gp-util namespaces really belong in the common dep as that dep is used to provide common utility fns across deps, which is the case here. I can make this change tomorrow if you'd like

Reply:
Moving gp-config and gp-util to the common dep sounds great! Yes, please make this change.

Reply:
I just [moved](https://github.com/logseq/logseq/pull/10793/commits/b7d9098e80aa40456aced3e8f06cc717b19fbfb5) both to the common dep since I need to move the outliner module to the outliner dep.

Reply:
Great!